YMCache/YMCachePersistenceController.m

```diff
@@ -239,12 +240,7 @@ - (BOOL)p_saveMemoryCacheUnsafe:(NSError * __autoreleasing *)error {
         return NO;
     }
     
-    // File protection is only supported on iOS.
-    // TODO: allow these to be provided by the client.
-    NSDataWritingOptions dataWritingOpts = NSDataWritingAtomic;
-#if TARGET_OS_IPHONE
-    dataWritingOpts |= NSDataWritingFileProtectionNone;
-#endif
+    NSDataWritingOptions dataWritingOpts = self.fileWritingOptions;
```

Comment:
Do you need this as a separate var here now that it's just a property on `self`?